site/source/docs/optimizing/Optimizing-Code.rst

```diff
@@ -182,6 +182,11 @@ Running by itself
 
 If you hit memory limits in browsers, it can help to run your project by itself, as opposed to inside a web page containing other content. If you open a new web page (as a new tab, or a new window) that contains just your project, then you have the best chance at avoiding memory fragmentation issues.
 
+Module Splitting
+----------------
+
+If your module is large enough that the time to download and instantiate it is noticeably affecting your application's startup performance, it may be worth splitting the module and deferring the loading of code that is not necessary to bring up the application. See :ref:`Module-Splitting` for a guide on how to do this.
+
```

Comment:
Is it maybe too early to put this in the general location here, if the feature is not fully stable? There is a warning on the page itself though so maybe that's enough.

Reply:
I mostly put this in because it seems that otherwise it might be difficult to find the documentation. Removing this makes sense, but I worry that I would forget to put it back in later. It seems better to just rely on warnings that this feature is experimental. I'll add a note about that here as well.